tests/types/index.ts

```diff
@@ -0,0 +1,68 @@
+export const enum ILinterVersion {
+  KnownGoodVersion = 1,
+  Latest,
+}
```

Comment:
Why assign `1` to `KnownGoodVersion`? I recommend using string unions instead of enums since they are much easier to work with (with almost no downsides)